src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -0,0 +1,315 @@
+              This field is required.
+            </span>
+          </div>
+        </div>
+
+        <!-- Size -->
+        <div class="form-group"
+             [ngClass]="{'has-error': (formDir.submitted || rbdForm.controls.size.dirty) && rbdForm.controls.size.invalid}">
+          <label translate
+                 class="control-label col-sm-3"
+                 for="size">Size
+            <span class="required"></span>
+          </label>
+          <div class="col-sm-9">
+            <input id="size"
+                   name="size"
+                   class="form-control"
+                   type="text"
+                   formControlName="size"
+                   placeholder="e.g., 10GB"
+                   cdDimlessBinary>
+            <span i18n
+                  class="help-block"
+                  *ngIf="(formDir.submitted || rbdForm.controls.size.dirty) && rbdForm.controls.size.hasError('required')">
+              This field is required.
+            </span>
+            <span i18n
+                  class="help-block"
+                  *ngIf="(formDir.submitted || rbdForm.controls.size.dirty) && rbdForm.controls.size.hasError('invalidSizeObject')">
+              You have to increase the size.
+            </span>
+          </div>
+        </div>
+
+        <!-- Features -->
+        <div class="form-group"
+             [ngClass]="{'has-error': (formDir.submitted || featuresFormGroups.dirty) && featuresFormGroups.invalid}"
+             formGroupName="features">
+          <label class="col-sm-3 control-label"
+                 for="features">
+            Features
+          </label>
+          <div class="col-sm-9">
+            <div class="checkbox checkbox-primary"
+                 *ngIf="!editing">
+              <input type="checkbox"
+                     id="default-features"
+                     name="default-features"
+                     formControlName="defaultFeatures">
+              <label i18n
+                     for="default-features">Use default features</label>
+            </div>
+            <div *ngIf="!featuresFormGroups.value.defaultFeatures">
+              <br *ngIf="!editing">
+              <div class="checkbox checkbox-primary"
+                   *ngFor="let feature of featuresList">
+                <input type="checkbox"
+                       id="{{ feature.key }}"
+                       name="{{ feature.key }}"
+                       formControlName="{{ feature.key }}">
+                <label for="{{ feature.key }}">{{ feature.desc }}</label>
+                <cd-helper *ngIf="feature.helperHtml"
+                           html="{{ feature.helperHtml }}">
+                </cd-helper>
+              </div>
+              <span i18n
+                    class="help-block"
+                    *ngIf="(formDir.submitted || featuresFormGroups.dirty) && featuresFormGroups.hasError('noFeatureSelected')">At least one feature has to be selected</span>
```

Comment:
Nit: not required to select at least one feature (0 is a valid feature bitmask).